pycbc/events/stat.py

```diff
@@ -28,33 +28,6 @@
 import numpy
 from . import events
 
-def get_statistic(option, files):
```

Comment:
If you'd like a dictionary, please this method implemented but have it use the dictionary internally. This method is used by other codes than your own, and I'd prefer not to break compatibility.